### ISSUE TITLE
Implement Definition Queries Business Logic

### ISSUE THREAD

**Issue by philipbaileynar** [Riverscapes/RiverscapesXML#297]
There's lots of interest in being able to filter out vector features in some of our feature classes in RAVE. See discussion #154 for how I have proposed doing this using definition queries. There are two unknowns before we can move forward:

1. How do definition queries differ between ArcGIS and QGIS and also ShapeFile v GeoPackage?
2. What are the implications for WebRAVE (can tippecanoe take an attribute filter as an argument)?

Please take a single feature class and filter the features across the four permutations described in option 1. Then repeat the process for a shapefile. I recommend using a string field because quotes are always messy (single versus double, escape sequences). Perhaps use the HUC code in our NHD data?

The goal is to fill in the following table with query strings.

|Storage Type|ArcGIS|QGIS|
|---|---|---|
|ShapeFile|`"watershed_id"='17060103'`|`"watershed_id"='17060103'`|
|GeoPackage|`"watershed_id"='17060103'`|`"watershed_id"='17060103'`|
|PostIGS|TODO|`"watershed_id"='17060103'`|

Advanced (Arcgis, QGIS and Postgis?):
- `"HUC8" = '01010002' and "HUC8" = '17060204'`
- `"HUC8" IN ('01010002','17060204','10190005')`
- `"HUC8" LIKE ('0101%')`

Tasks
- [x] [Implement in QRAVE](https://github.com/Riverscapes/QRAVEPlugin/issues/56)
- [ ] [Implement in ArcRAVE](https://github.com/Riverscapes/RaveAddIn/issues/172)

**Comment by joewheaton** [comment]
The issue as to whether or not we can deal with this in WebRAVE is a separate issue... and yes @MattReimer could be tricky.

**Comment by joewheaton** [comment]
This is done.
